controllers/util/k8sutil.go

```diff
@@ -53,7 +53,8 @@ const (
 	OdfInfoNamespacedNameClaimName      = "odfinfo.odf.openshift.io"
 	ExitCodeThatShouldRestartTheProcess = 42
 
-	BlockPoolMirroringTargetIDAnnotation = "ocs.openshift.io/mirroring-target-id"
+	BlockPoolMirroringTargetIDAnnotation     = "ocs.openshift.io/mirroring-target-id"
+	RequestFailoverMaintenanceModeAnnotation = "ocs.openshift.io/request-failover-maintenance-mode"
```

Comment:
At this point in time we decided not to differentiate between the modes and even if we do we will do that via the value of the annotation not the key
```suggestion
	RequestMaintenanceModeAnnotation = "ocs.openshift.io/request-maintenance-mode"
```